products/apache-nifi.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Apache NiFi
+category: server-app
+tags: apache java-runtime
+iconSlug: apachenifi
+permalink: /apache-nifi
+alternate_urls:
+-   /nifi
+changelogTemplate: "https://cwiki.apache.org/confluence/display/NIFI/Release+Notes#ReleaseNotes-Version__LATEST__"
+eolColumn: Support
+releaseDateColumn: true
+
+identifiers:
+-   repology: nifi
+
+auto:
+  methods:
+  -   git: https://github.com/apache/nifi.git
+
+releases:
+-   releaseCycle: "1.27.0"
```

Comment:
```suggestion
-   releaseCycle: "1.27"
```

If Apache Nifi follows semver, 1.27 is the release. This comment applies to all releases.